tiddlers/TiddlyWiki.tid

```diff
@@ -4,6 +4,6 @@ tags: .
 title: TiddlyWiki
 type: text/vnd.tiddlywiki
 
-英文官网：https://tiddlywiki.com/
+Site officiel en anglais：https://tiddlywiki.com/
 
-中文文档：https://zh-hans.tiddlyhost.com/
```

Comment:
If there is a TiddlyWiki document in French, this should be changed to it. Otherwise, just delete it.